frontend/test/metabase/scenarios/account/notifications/notifications.cy.spec.js

```diff
@@ -14,6 +14,10 @@ const questionDetails = {
   },
 };
 
+const dashboardDetails = {
+  name: "Dashboard",
+};
```

Comment:
If you needed any generic dashboard name, then this can safely be omitted. We already provide a fallback name for the dashboard.

In that case you don't pass `dashboardDetails` to the `cy.createQuestionAndDashboard()`. Hope this helps :)

Reply:
And yeah - the fallback name is "Test Dashboard".

Reply:
Fixed, thanks!